kumascript/macros/CSSSyntax.ejs

```diff
@@ -1,442 +1,324 @@
 <%
-/*
-  Displays the syntax of a CSS property or descriptor
-
-  $0 - property/descriptor name - defaults to the slug name
-  $1 - @-rule - defaults to the @-rule within the URL
-*/
```

Comment:
Even if these parameters were obsolete (are they?), it would be good to keep a comment at the top, explaining what the macro does.